README.md

```diff
@@ -261,8 +261,8 @@ the ansible role.
 
 #### `type: wireless`
 
-The `wireless` type supports WPA-PSK (password) authentication and WPA-EAP (802.1x)
-authentication.
+The `wireless` type supports WPA-PSK (password) authentication, WPA-EAP (802.1x)
+authentication, and OWE authentication.
```

Comment:
OWE is not an authentication method, it is only about encryption.

Reply:
Changed into: 
```
The `wireless` type supports WPA-PSK (password) authentication, WPA-EAP (802.1x)
authentication, and Enhanced Open (OWE).
```